src/FilterMonitor.js

```diff
@@ -10,10 +10,11 @@ export default class FilterMonitor extends Component {
     blacklist: PropTypes.array
   };
 
-  isFiltered(action) {
+  isFiltered(actionType) {
+    var type = actionType || '';
```

Comment:
It shouldn't. Unfortunately, I don't have a project with Redux DevTools to test it, and this monitor is not part of our extension anymore. If you have any details let me know.